### DISCUSSION TITLE
GHF for H2 molecule, singlet, triplet states

### DISCUSSION THREAD

**Discussion by Physfock** [pyscf/pyscf#2615] [Q&A]
I am new to quantum chemistry and would be grateful for some advice on how to calculate the energy of the triplet state of the H₂ molecule using the GHF method. At present, the calculation always converges to the singlet state.

**Comment by sunqm** [comment]
```
mol = pyscf.M(atom='H 0 0 -0.5; H 0 0 0.5', basis='cc-pvtz', spin=2)
mf = mol.UHF().run()
mf = mf.to_ghf()
```
